### ISSUE TITLE
Training and testing datasets to download 

### ISSUE THREAD

**Issue by pflashgary** [conrad-blucher-institute/FogNet#4]
Thanks for making this open source. Can we ask for the training/testing datasets including the GRIB files and observations?
The use of code and data will be referenced properly. 

On a separate note, can we contribute to the repository through issues and pull requests?

**Comment by ekrell** [comment]
Thank you for your interest! The files are big, we are currently discussing where is best for us to host them. Hope to get them available very soon. 

And any contributions are very welcome, thanks

**Comment by pflashgary** [comment]
Would it work to extract only the variables used in the paper from the GRIB files and save them as a compressed binary format? This is just a simple suggestion. We'd like to develop your network to produce probability maps.  

Thanks

**Comment by pflashgary** [comment]
Before I close this issue I was thinking, could we use this [archive](https://www.ready.noaa.gov/data/archives/nam12/README.TXT)? If yes:

- Would it be useful to provide the link on README and also
- Maybe the observational data could reside on the repo? Or another link to where to download?

**Comment by ekrell** [comment]
Sorry for the delay. I have not been ignoring this, working on setting some things up. Slowly because I came on the project much later and was not part of the original data acquisition. The persons that are, are super busy getting ready for AMS2022 presentations. 

The data for FogNet is a little complex, because our input rasters combine NAM and [MUR SST data](https://podaac.jpl.nasa.gov/MEaSUREs-MUR). 

But, we now have an FTP server to provide the data: https://gridftp.tamucc.edu/fognet/

So far, I have added the **processed predictor rasters** for 12 and 24-hour lead times: https://gridftp.tamucc.edu/fognet/
You will see the predictors, targets, etc in there, and should be able to at least run FogNet. I've updated the README with more information. 

We also have a bunch of scripts that were used to download the data and generate these rasters. I hope to have them to you soon.
